src/exoscale/compute/api/http.clj

```diff
@@ -65,9 +65,12 @@
 
 (defn find-payload
   [resp]
-  (let [ks (set (keys resp))]
-    (if (and (= 2 (count ks)) (contains? ks :count))
-      (-> resp (dissoc :count) vals first)
+  (let [ks (set (keys resp))
+        payload (-> resp (dissoc :count) vals first)]
+    (if (and (not= 0 (count ks))
```

Comment:
```suggestion
    (if (and (pos? (count ks))
```

Reply:
If you think it's easier to read.  Having `(seq ks)` instead of `(not= 0 (count ks))` would work too.